pkgs/tools/cd-dvd/cue2pops/default.nix

```diff
@@ -5,29 +5,30 @@
 
 stdenv.mkDerivation {
   pname = "cue2pops";
-  version = "unstable-2018-01-04";
+  version = "0-unstable-2023-01-15";
 
   src = fetchFromGitHub {
     owner = "makefu";
     repo = "cue2pops-linux";
-    rev = "541863adf23fdecde92eba5899f8d58586ca4551";
-    sha256 = "05w84726g3k33rz0wwb9v77g7xh4cnhy9sxlpilf775nli9bynrk";
+    rev = "3f2be6126bd50dfe6b998bc8926f88ce9139c19a";
+    hash = "sha256-7rgYvqeH8ZDI8Vc5vnjIhe3Ke0TE1q/iFHEqucanhUM=";
   };
 
   dontConfigure = true;
 
-  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
-
   installPhase = ''
+    runHook preInstall
     install --directory --mode=755 $out/bin
     install --mode=755 cue2pops $out/bin
+    runHook postInstall
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Convert CUE to ISO suitable to POPStarter";
     homepage = "https://github.com/makefu/cue2pops-linux";
-    maintainers = with maintainers; [ AndersonTorres ];
-    platforms = platforms.all;
+    license = lib.licenses.unfree;
```

Comment:
https://github.com/ErikAndren/cue2pops-mac/issues/2

> License is very unclear. I cannot recall where I got the source from. Likely some forum post somewhere. Sorry

`unfree` it is 🙃

Reply:
Another good thing to do is to ask upstream about licensing.
But not today, let's update this first.

Reply:
It seems like it’s passed down through random forum posts to multiple GitHub forks without any clear authorship or licence information. If anyone knows PlayStation community lore then maybe they could work on tracking down the original author and every contributor since and getting them to agree on a licence, but it seems impractical.